lib/wire-protocol.d.ts

```diff
@@ -45,29 +45,64 @@ export interface QueryRequestHeaders {
    */
   traceparent?: string;
 }
-/**
- * A response to a query.
- * @remarks
- * The QueryResponse is type parameterized so that you can treat it as a
- * a certain type if you are using Typescript.
- */
-export interface QueryResponse<T> {
+export declare type QueryInfoStats = {
+  /** The amount of Transactional Compute Ops consumed by the query. */
+  compute_ops: number;
+  /** The amount of Transactional Read Ops consumed by the query. */
+  read_ops: number;
+  /** The amount of Transactional Write Ops consumed by the query. */
+  write_ops: number;
+  /** The query run time in milliseconds. */
+  query_time_ms: number;
+  /** The amount of data read from storage, in bytes. */
+  storage_bytes_read: number;
+  /** The amount of data written to storage, in bytes. */
+  storage_bytes_written: number;
+  /** The number of times the transaction was retried due to write contention. */
+  contention_retries: number;
+};
+export declare type QueryInfo = {
```

Comment:
`QueryInfo` comes from the openAPI spec. In general I am on board with `QueryMetadata` but I think it would be better to keep in sync with the openAPI spec.

Reply:
oh if that's from open API let's go with that.